src/Restaurant.js

```diff
@@ -1,28 +1,26 @@
 import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
 
 const Restaurant = () => {
   // array destructuring
   // setOrders is a function that allows me to modify the value of the orders variable
   // 0 is the initial value
   const [orders, setOrders] = useState(0);
 
+  function OrderOne() {
+    setOrders(orders + 1);
```

Comment:
Another way of doing this would be `setOrders((orders) => orders + 1)`, which is a slightly more robust way of doing this for [reasons laid out in the lesson for React week 2](https://syllabus.codeyourfuture.io/react/week-2/lesson#setting-state-based-on-previous-state).

Reply:
Thank you DOuglas, I totally forgot about this. I understand why now :)